docs/backup_troubleshooting.md

```diff
@@ -12,7 +12,7 @@ Another good way to check if there is activity is the XOA VM stats view (on the
 
 ### VDI chain protection
 
-This means your previous VM disks and snapshots should be "merged" (*coalesced* in the XenServer world) before we can take a new snapshot. This mechanism is handled by XenServer itself, not Xen Orchestra. But we can check your existing VDI chain and avoid creating more snapshots than your storage can merge. Otherwise, this will lead to catastrophic consequences. Xen Orchestra is the **only** XenServer/XCP backup product dealing with this.
+Backup jobs regularly delete snapshots. When a snapshot is deleted, either manually or via a backup job, it triggers the need for Xenserver to coalesce the VDI chain - to merge the remaining snapshots. This means generally we cannot take too many new snapshots on said VM until Xenserver has finished running a coalesce job on the VDI chain. This mechanism and scheduling is handled by XenServer itself, not Xen Orchestra. But we can check your existing VDI chain and avoid creating more snapshots than your storage can merge. If we don't, this will lead to catastrophic consequences. Xen Orchestra is the **only** XenServer/XCP backup product that takes this into account and offers protection.
```

Comment:
> to merge the remaining snapshots

I think this is unclear, here's my explanation of what's happening:

When a VDI is cloned (like a snapshot), XenServer creates an special unmanaged VDI called *base copy* which hold the data in common between the original VDI and the cloned VDI.

This process will happen again every time one of the VDIs is cloned and this will create a chain of VDIs and *base copies*. Long chains are problematic because they impact IO performance, therefore XenServer limits the length to a certain size.

When one of the VDIs is deleted, the other VDI and the *base copy* will be coalesced by XenServer. This process can take some time, maybe more time than we create new *base copies* by creating/deleting snapshots of this VDI.

That's why XO detects this situation and prevents new automatic snapshots from being created when there are some *base copies* that need to be coalesced.

Reply:
hmm, I worry that might be too in-depth/technical for average users, but you would know better than I

Reply:
I'm not saying that it must be so technical, just that XenServer is not merging the remaining snapshots but the base copies left by the deletion of a snapshot :slightly_smiling_face:

Reply:
Ah, got it. Will clarify the explanation as well as add a hint about migrating the VDI then request a new review